static/bidder-info/zeta_global_ssp.yaml

```diff
@@ -0,0 +1,20 @@
+endpoint: https://ssp.disqus.com/bid/prebid-server?sid=GET_SID_FROM_ZETA&shortname=GET_SHORTNAME_FROM_ZETA
+endpointCompression: gzip
+maintainer:
+  email: DL-Zeta-SSP@zetaglobal.com
```

Comment:
@abermanov-zeta I have sent a verification email to this address. Please reply to that email to complete the maintainer email verification process.

Reply:
email verification completed!

Reply:
Thank you